common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -54,6 +54,14 @@ def create_modulestore_instance(
     )
 
 
+def mock_tab_from_json(tab_dict):
```

Comment:
I was consolidating them and missed the one in `test_cross_modulestore_import_export`. It's not meant to be duplicated.